resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/submit/submitsteps/DriverServiceBootstrapStepSuite.scala

```diff
@@ -147,7 +128,7 @@ private[spark] class DriverAddressConfigurationStepSuite
     } catch {
       case e: Throwable =>
         assert(e.getMessage ===
-            s"requirement failed: ${DriverAddressConfigurationStep.DRIVER_HOST_KEY} is" +
+            s"requirement failed: ${DriverServiceBootstrapStep.DRIVER_HOST_KEY} is" +
```

Comment:
this should be indented 2 spaces not 4 (same for the ones above).
http://spark.apache.org/contributing.html

Reply:
Done.